apps/namadillo/src/atoms/integrations/services.ts

```diff
@@ -0,0 +1,54 @@
+import { OfflineSigner } from "@cosmjs/launchpad";
+import { coin, coins } from "@cosmjs/proto-signing";
+import { SigningStargateClient } from "@cosmjs/stargate";
+import BigNumber from "bignumber.js";
+
+export type IBCTransferParams = {
+  signer: OfflineSigner;
+  sourceAddress: string;
+  destinationAddress: string;
+  amount: BigNumber;
+  token: string;
+  channelId: string;
+  memo?: string;
+};
+
+export const submitIbcTransfer =
+  (transferParams: IBCTransferParams) =>
+  async (rpc: string): Promise<void> => {
+    const {
+      signer,
+      sourceAddress,
+      destinationAddress,
+      amount,
+      token,
+      channelId,
+      memo,
+    } = transferParams;
+
+    const client = await SigningStargateClient.connectWithSigner(rpc, signer, {
+      broadcastPollIntervalMs: 300,
+      broadcastTimeoutMs: 8_000,
+    });
+
+    const fee = {
+      amount: coins("0", token),
+      gas: "222000",
+    };
+
+    const response = await client.sendIbcTokens(
+      sourceAddress,
+      destinationAddress,
+      coin(amount.toString(), token),
```

Comment:
This will be a pain, but somewhere we're going to need to convert the amount the user input to a different denomination for sending (I think). For example, the user inputs 2 ATOM, and we will need to convert it to 2000000 uatom.